src/steamship/app/app.py

```diff
@@ -72,7 +72,7 @@ def post(path: str, **kwargs):
     return endpoint(verb=Verb.POST, path=path, **kwargs)
 
 
-class App:
+class App:  # TODO (enias): Should be renamed to Invocable -> Solves redefinition of app
```

Comment:
Perhaps in a base class thought? 

From the user perspective, might make more sense to have `Package` and `Plugin` derive from `Invocable`